hardhat.config.ts

```diff
@@ -60,6 +60,12 @@ const config: HardhatUserConfig = {
       chainId: 11155111,
       accounts: process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : [],
     },
+    mumbai: {
```

Comment:
```suggestion
    polygonMumbai: {
```
This should match with etherscan keys. And I think it will be clearer.  Or let's set mumbai on etherscan keys.

Reply:
For some reason, setting etherscan kay to `mumbai` fails as it requires `polygonMumbai` there. But we could try applying your suggestion and see if everything works as expected.

This is the only network that I've seen such behavior, but the mismatched named keys work for some reason.

Reply:
It tried locally in both EVM and marketplace with `polygonMumbai` in both in worked well so I'll apply it :smile: